src/racer/codecleaner.rs

```diff
@@ -123,26 +133,58 @@ impl<'a> CodeIndicesIter<'a> {
         self.code()
     }
 
-    fn string(&mut self) -> SourceByteRange {
+    fn string(&mut self, str_type: StringType) -> SourceByteRange {
         let src_bytes = self.src.as_bytes();
         let mut pos = self.pos;
-        if pos > 1 && src_bytes[pos-2] == b'r' {
-            // raw string (eg br"\"): no escape
-            match src_bytes[pos..].iter().position(|&b| b == b'"') {
-                Some(p) => pos += p+1,
-                None    => pos = src_bytes.len()
+        match str_type {
+            StringType::Raw(level) => {
+                // raw string (eg br#"\"#)
+                // detect corresponding end(if start is r##", ##") greedily
```

Comment:
`if start is r##", ##"` - should the second one of those be `"##`, or am I misunderstanding this comment?